### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by ghost**
adding missing README for module

### CODE REVIEW

File: gateway/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# javascript.client
```
Comment (vjwang) [inline gateway/README.md]: Will you also link to this README from the parent-level README?

Comment (ghost) [inline gateway/README.md]: I don't intend to link subs to parents, no. They have to click the parent to see this sub.

Comment (vjwang) [inline gateway/README.md]: That's what I meant, but I see that I mistook this for a parent README. Consider my original comment resolved.

**Comment by vjwang** [comment]
@michaelcretzman Please resolve the conflicts.

**Comment by ghost** [comment]
@vjwang , I think you can merge this now

**Comment by vjwang** [comment]
@michaelcretzman Still can't merge it.

**Comment by ghost** [comment]
looks like it doesn't matter since the change made it in somehow:
https://github.com/kaazing/javascript.client

Closing this PR

**Pull request closed without merging**